server/message_hooks.go

```diff
@@ -515,9 +523,11 @@ func (p *Plugin) SendChat(srcUser string, usersIDs []string, post *model.Post) (
 		attachment, err = client.UploadFile("", "", fileName+"_"+fileInfo.Id+fileExtension, int(fileInfo.Size), fileInfo.MimeType, bytes.NewReader(fileData), chat)
 		if err != nil {
 			p.API.LogWarn("Error in uploading file attachment to MS Teams", "error", err)
+			p.metricsService.ObserveFilesCount(actionCreated, actionSourceMattermost, directMessageTrue, discardedReasonUnableToUploadFileOnTeams, increaseFileCountByOne)
```

Comment:
Any specific reason why we are using string for True and false ?

Reply:
Yes, the `prometheus.Labels` is a type of `map[string]string` so we need to have it as a string.